predictable-app-updating.md

```diff
@@ -4,34 +4,41 @@ Author: Dibyajyoti Pal (dibyapal@chromium.org)
 
 # **Introduction**
 
-This explainer proposes a way to have PWAs fully update their identities in a safe and resourceful manner, to further bridge the gap between PWAs and native apps. This is done by updating the [manifest spec](https://www.w3.org/TR/appmanifest/#web-application-manifest) to have a specific field for making updates more deterministic by the developer, and for providing a consistent experience. The proposal attempts to do so in a way that:
+This explainer proposes a way to have PWAs fully update their identities in a safe and resourceful manner, to further bridge the gap between PWAs and native apps. This is done by updating the [manifest spec](https://www.w3.org/TR/appmanifest/#web-application-manifest) to have a specific field for making updates more deterministic by the developer, and for providing a consistent experience across different user agents. The proposal attempts to do so in a way that:
 
 1. Uses less resources, making network usage more efficient.  
 2. Prevent user confusion by showing the update UX less often.
+3. Allows user agents who haven't yet implemented manifest updating to do so in a different way.
+
+One thing to note here is that installed PWAs are inherently different from how the rest of the Web Platform works. While the origin is the security boundary of the Web Platform, the installed experience has [different security parameters](https://www.w3.org/TR/appmanifest/#dfn-security-sensitive-members).
 
 # **Background**
 
-[PWAs](https://web.dev/explore/progressive-web-apps) are app experiences built on the web, and like native apps, they support updating themselves. Developers have the option of changing any field in the manifest (including the [security sensitive fields](https://www.w3.org/TR/appmanifest/#dfn-security-sensitive-members)), and the user agent can then apply the changes in a way as defined in the [manifest updating spec](https://www.w3.org/TR/appmanifest/#updating). One of the ways Chromium does it is by showing the update dialog, as seen below under the `Chromium PWA update detection` section. This is done to prevent known [phishing risks](#phishing).
+[PWAs](https://web.dev/explore/progressive-web-apps) are app experiences built on the web, and like native apps, they support updating themselves. Developers have the option of changing any field in the manifest (including the [security sensitive fields](https://www.w3.org/TR/appmanifest/#dfn-security-sensitive-members)), and the user agent can then apply the changes in a way as defined in the [manifest updating spec](https://www.w3.org/TR/appmanifest/#updating). For example, in an effort to prevent [phishing risks](#phishing), some user agents like Chromium shows an update dialog to the user.
```

Comment:
mention that they SHOULD show UX.  This line from the spec is a requirement for user agents!

```
Instead, user agents SHOULD present changes to [security-sensitive members](https://www.w3.org/TR/appmanifest/#dfn-security-sensitive-members) with appropriate management options, so the user can make an informed decision about updating the web application.
```